### ISSUE TITLE
Default bind to higher port in www component

### ISSUE THREAD

**Issue by wkoot** [ICTU/quality-time#9209]
- [ ] Change the default `PROXY_PORT` to `8080`, so we don't bind a privileged port. Since this will remove backwards compatability and changes to docker-compose, it requires a major release.
- [ ] Remove `CAP_NET_BIND_SERVICE` from the proxy Dockerfile and helm deployment. This will allow for easier deployments on environments that have stricter rules.
- [ ] Update deployment documentation
- [ ] Update software documentation
- [ ] Update versioning policy

**Comment by fniessink** [comment]
Would there a way to still use port 80 when deploying locally?

**Comment by wkoot** [comment]
Of course, the port mapping can be 8080 inside the container to 80 outside the container (i.e. localhost)

**Comment by wkoot** [comment]
https://github.com/ICTU/quality-time/pull/12986 probably solves this for helm deployments
